src/components/views/read/cvForm/Education.tsx

```diff
@@ -97,6 +129,12 @@ const Education: FC<Props> = ({
   };
 
   const handleRemove = (cardId: string): void => {
+    const objToDelete = appDataArray.filter(
```

Comment:
This already exists under the `educationInfoObj` variable, why re-define it here ?